crates/cargo-test-macro/src/lib.rs

```diff
@@ -302,7 +302,37 @@ fn check_command(command_path: &Path, args: &[&str]) -> bool {
 }
 
 fn has_command(command: &str) -> bool {
-    check_command(Path::new(command), &["--version"])
+    use std::env::consts::EXE_EXTENSION;
+    let Some(paths) = std::env::var_os("PATH") else {
+        return false;
+    };
+    // ALLOWED: For testing cargo itself only.
+    #[allow(clippy::disallowed_methods)]
```

Comment:
Thought so, CI says this is on the wrong line.  It should be on `var_os`

Reply:
🤦🏾‍♂️

Fixed!